types/react-flatpickr/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for react-flatpickr 3.2
+// Project: https://github.com/coderhaoxin/react-flatpickr
+// Definitions by: begincalendar <https://github.com/begincalendar>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import { Component } from 'react';
+import { EventCallback, Options } from 'flatpickr';
+
+interface DateTimePickerProps {
+    defaultValue?: string;
+    options?: Options;
+    onChange?: EventCallback;
+    value?: string;
+}
+
+declare class DatePicker extends Component<DateTimePickerProps, void> {}
```

Comment:
Just add `export` in front of `interface DateTimePickerProps`. Since this is a module declaration everything in it is implicitly exported, the lint rule just asks to make it explicit.